test/test-layout.js

```diff
@@ -0,0 +1,43 @@
+var should = require('should');
+
+var lib = './../lib/';
+
+var Graph = require(lib + 'model/graph');
+var Node = require(lib + 'model/node');
+var Port = require(lib + 'model/port');
+var CardinalPortPosition = require(lib + 'model/cardinalPortPosition');
+var Connection = require(lib + 'model/connection');
+
+var LayoutEngineV1 = require(lib + 'layout/dummyLayoutV1');
+var LayoutEngineV2 = require(lib + 'layout/dummyLayoutV2');
+
+var generateGraph = function() {
+  // the world's simplest graph
+  var port = new Port('a+a', 'input', new CardinalPortPosition(50, 'N'));
+  var node = new Node('a', [port], []);
```

Comment:
If you don't mind, @johnwobrien, please hand off this PR to me instead of merging it when merge time comes. I'd like to enact the naming changes before it goes into develop.